sle12/templates/csv/services_disabled.csv

```diff
@@ -3,7 +3,6 @@ abrtd,abrt,
 acpid,,
 atd,at,
 autofs,,
-avahi-daemon,avahi,
```

Comment:
This will also stop generation of templated OVAL and Bash for service_avahi-daemon_disabled.

Reply:
Workaround I see are, also moving OVAL and Bash to shared folder, or add another column limiting generation via templates for target languages.

Reply:
I see, another possibilities are:
* Make the avahi remediation to use the template. It would lose the "disable socket" functionality that way.
* Make service disable remediations more generic, so they can optionally contain socket disable Ansible remediation.